src/docs/asciidoc/data-access.adoc

```diff
@@ -2730,17 +2730,15 @@ The following query finds and populates a single domain object:
 [source,java,indent=0,subs="verbatim,quotes",role="primary"]
 .Java
 ----
+	RowMapper<Actor> actorRowMapper = (resultRest, rowNum) -> {
```

Comment:
Please inline the lambda expression as was previously the case with the anonymous inner class, since that then aligns with the explanatory text in the following paragraph.